products/pubsub/terraform.yaml

```diff
@@ -124,6 +124,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       expirationPolicy.ttl: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'comparePubsubSubscriptionExpirationPolicy'
+      retryPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+      retryPolicy.maximumBackoff: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
It doesn't make sense to set `default_from_api` and a `default_value` on the same field. `default_from_api` is specifically about the behavior when the field is not set by the user, and setting a default means we'll always send something.

I'd recommend just using `default_value`, since it'll show up when the user runs `terraform plan`.